README.md

```diff
@@ -51,6 +51,79 @@ Be patient when OpenOCD is flashing. Typically, it takes about 55 seconds.
 Use `make help` to get detailed build targets.
 
 
+# 64-bit System Support
+
+## 1. GNU Toolchain
+
+We extensively rely on the GNU toolchain to streamline our development process. At its core, we utilize the "Make" utility as the orchestration engine, automating the compilation and building of our source code. With a meticulously crafted Makefile, we define rules and dependencies necessary for compiling, linking, and generating binary images from our codebase.
+
+
+Open a terminal and update your package repository to ensure you have the latest information about available packages.
+
+```bash
+    sudo apt update
+```
+Use the package manager to install the GNU Toolchain, which includes tools like gcc, g++, and binutils.
+
+```bash
+    sudo apt-get install build-essential
+```
+
+After the installation is complete, you can verify it by checking the installed version of GCC.
+
+```bash
+    gcc --version
+```
+
+## 2. Compiler
+Since the ARM-2010q1 toolchain is intended for 32-bit systems and if your system is 64-bit, you'll need to enable 32-bit support and install necessary libraries. Execute the following commands:
+```bash
+    sudo dpkg --add-architecture i386
+    sudo apt-get update
+    sudo apt-get install libc6:i386 libncurses5:i386 libstdc++6:i386
+```
+
+## 3. ST_Link Programmer
```

Comment:
Should be `STLINK Tools` as [stlink](https://github.com/stlink-org/stlink) indicates.